.builds/linux.yml

```diff
@@ -0,0 +1,36 @@
+image: archlinux
+
+packages:
+  - pkg-config
+  - cmake
+  - make
+  - freetype2
+  - fontconfig
+  - libxcb
+
+sources:
+  - https://github.com/alacritty/alacritty
+
+environment:
+  PATH: /home/build/.cargo/bin:/usr/bin/
+
+tasks:
+  - rustup: |
+      curl https://sh.rustup.rs -sSf | sh -s -- -y --profile minimal --default-toolchain stable
+  - test: |
+      cd alacritty
+      cargo build
```

Comment:
what's the point of 'cargo build', just 'cargo test' should be sufficient?

Reply:
I think it's useful to do both, since one will use the previous artifacts anyways and you know if just the tests fail, or if the actual build fails already.

Reply:
That not entirely true, it'll relink iirc, so it'll add more build time than it should. Though, maybe having separate status justifies building more.